Abennat/DLS001.xml

```diff
@@ -1,18 +1,19 @@
-<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
 schematypens="http://relaxng.org/ns/structure/1.0"?>
-<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="DLS001" xml:lang="en" type="mss">
     <teiHeader>
         <fileDesc>
             <titleStmt>
                 <title>Dǝggʷā</title>
+                <title>Dǝgʷā</title>
```

Comment:
dear Addisie, you do not need a second line for the title, line 9 (Dǝggʷā) was correct, I had already corrected it